blockchain/difficulty.go

```diff
@@ -244,7 +244,8 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 			// Return minimum difficulty when more than the desired
 			// amount of time has elapsed without mining a block.
 			reductionTime := b.chainParams.MinDiffReductionTime
```

Comment:
I changed all of the other references and the math is now all performed in terms of seconds.